File: apps/chat/chat-server/src/main/java/com/example/chat/config/OpenApiConfig.java
package com.example.chat.config;

import io.swagger.v3.oas.models.OpenAPI;
import io.swagger.v3.oas.models.info.Info;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class OpenApiConfig {
	@Bean
	public OpenAPI chatServerOpenAPI() {
		return new OpenAPI().info(new Info().title("Chat Server API").version("v1").description("채팅 서버 API (message + system 통합)"));
	}
}

File: apps/chat/system-server/src/main/kotlin/com/example/chat/system/config/OpenApiConfig.kt
package com.example.chat.system.config

import io.swagger.v3.oas.models.OpenAPI
import io.swagger.v3.oas.models.info.Info
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class OpenApiConfig {

	@Bean
	fun openAPI(): OpenAPI {
		return OpenAPI()
			.info(
				Info()
					.title("Chat System API")
					.version("1.0")
					.description("채팅 시스템 관리 API")
			)
	}
}
